### ISSUE TITLE
Variant builds in travis-ci

### ISSUE THREAD

**Issue by arekinath** [arekinath/PivApplet#39]
We should run the `build.rb` script in travis-ci to generate all the CAP files for the different variants automatically.

**Comment by mistial-dev** [comment]
Travis-ci.org went commercial shortly after this issue was created, and then went read-only.

@arekinath would you like me to take a look at building under Github Actions?

**Comment by arekinath** [comment]
@mistial-dev yeah, we should move it over to github actions I guess. I need to move a bunch of my other repos over as well at some point, I was being lazy because I don't often have to change the build settings, so things were mostly still working through most of the drama.
